packages/stacked_services/lib/src/snackbar/snackbar_service.dart

```diff
@@ -122,26 +122,26 @@ class SnackbarService {
     );
 
     final getBar = GetBar(
-      titleText: Text(
+      title: title,
+      message: message,
+      titleText: snackbarConfig?.titleColor != null
+          ? Text(
         title,
         style: TextStyle(
-          color: snackbarConfig?.titleColor ??
-              snackbarConfig?.textColor ??
-              Colors.white,
-          fontWeight: FontWeight.w800,
-          fontSize: 16,
-        ),
-      ),
-      messageText: Text(
+            color: snackbarConfig.titleColor,
```

Comment:
When text color is set it won't take effect anymore. This now forces the developer to always supply a titleColor and a messageColor which is not how we're using it at the moment. This also removes the default styling of a grey snackbar with white text.

Is there any reason you would remove existing functionality if it causes no problems?

Please make sure when the user does not supply a titleColor the general textColor is used. If the user does not supply a textColor then white is used. The same goes for the messageColor